hostProviders/aws/postprovision/lsf/example_user_data.sh

```diff
@@ -57,9 +55,16 @@ fi
 #echo "lsfdev1.test.com" > $LSF_ENVDIR/hostregsetup
 #lsreghost -s $LSF_ENVDIR/hostregsetup
 
+
+# Install LSF as a service and start up
+${LSF_TOP}/10.1/install/hostsetup --top="${LSF_TOP}" --boot="y" --start="y" --dynamic
```

Comment:
aws and azure has no "  2>&1 >> $logfile" after --dynamic, while last 3 have. Please add for aws and azure